src/pages/demo/button/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { v4 as uuidv4 } from 'uuid'
+import { drupalClient } from '@/utils/drupalClient'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+import { DrupalNode } from 'next-drupal'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Operators from 'drupal-jsonapi-params/lib/operators'
+import Container from '@/components/container'
+import Button from '@/components/paragraph/button'
+import _ from 'lodash'
+
+interface ButtonPageProps {
+  buttons: DrupalNode[]
+}
+
+const ButtonPage = ({ buttons }: ButtonPageProps) => {
+  console.log('buttons ', buttons)
+  if (!buttons) buttons = []
+
+  // TODO: Remove once the DrupalJsonApiParams can be implemented
+  const filteredButtons = _.filter(buttons, function (button) {
+    return (
+      button.field_button_label !== null && button.field_button_link !== null
+    )
+  })
```

Comment:
Not a big deal but I think we can avoid and just filter out anything missing a label if don't get the DrupalJsonApiParams working

```
  const filteredButton = buttons.filter(
    ({ field_button_label }) => field_button_label !== null
  )
```